content/background.html

```diff
@@ -281,7 +287,7 @@ <h2> Others </h2>
 <h3>Programming Knowledge</h3>
 <p class="half">
   <span class="big">
-  5.6% of students did not know how to program prior to starting University.
+  5.6% of respondents did not know how to program prior to starting University.
```

Comment:
```suggestion
  5.6% of respondents did not know how to program prior to starting university.
```

i don't think university should be capped in this case because we're talking about in general...? (even if we are referring to UW)

otherwise, we could say UWaterloo (to keep the caps)